content/quickstart.py

```
@@ -5,6 +5,159 @@
+        return False
+        
+    cols = render_download_box(
+        container_key="windows_download_container",
+        title="FLASHApp for Windows",
+        description="FLASHApp is best enjoyed online but you can download an offline version for Windows systems below.",
+        icon=""
+    )
+    
+    # Center the download button
+    with cols[1]:
+        with open("OpenMS-App.zip", "rb") as file:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)